pages/integrations/output_framework.rst

```diff
@@ -0,0 +1,142 @@
+.. _output_framework:
+
+***************************
+Enterprise Output Framework
+***************************
+
+The Enterprise Output Framework provides the ability to forward data from your Graylog 
+cluster to external systems using a variety of network transport methods and payload 
+formats. In addition, you can configure Framework-based Outputs to use 
+:doc:`Processing Pipelines <../pipelines/pipelines>` to filter, modify, and enrich the 
+outbound messages.
+
+.. note:: This is an Enterprise Integrations feature and is only available since Graylog 
+          version 3.3.3, thus an Enterprise license is required. See the 
+          :doc:`Integrations Setup <setup>` page for more info.
+
+Enterprise Framework Outputs
+----------------------------
+
+The Enterprise Output Framework provides a number of new Outputs for the various network 
+transport types. All of these Outputs first write messages to an on-disk journal in the 
+Graylog cluster.  Messages stay in the on-disk journal until the Output is able to 
+successfully send the data to the external receiver.
+
+Once the messages have been written to the journal, they are optionally run through a 
+processing pipeline so unwanted messages can be filtered out and the remaining messages can 
+be modified or enriched with additional data.
```

Comment:
Maybe re-word this a bit:   (Just so the filter is not the first thing :))

processing pipeline to modify or enrich logs with additional data, transform the message contents or filter out any logs before sending.

Reply:
Will do